src/adv.py

```diff
@@ -49,3 +55,60 @@
 # Print an error message if the movement isn't allowed.
 #
 # If the user enters "q", quit the game.
+
+
+cmds = [
+    "n", "s", "e", "w", 
+    "N", "S", "E", "W",
+    "North", "South", "East", "West",
+    "north", "south", "east", "west"]
+
+def direction(player, cmd):
+    if cmd == "n" or cmd == "N" or cmd == "North" or cmd == "north":
+        if player.current_room.n_to:
+            dir = "Go North!"
+            player.current_room = player.current_room.n_to
+        else:
+            return print('\x1b[0;33;40m' +"I'm sorry you can't go North." + '\x1b[0m')
+    elif cmd == "s" or cmd == "S" or cmd == "South" or cmd == "south":
+        if player.current_room.s_to:
+            dir = "Go South!"
+            player.current_room = player.current_room.s_to
+        else:
+            return print('\x1b[0;33;40m' + "I'm sorry you can't go South." + '\x1b[0m')
+    elif cmd == "e" or cmd == "E" or cmd == "East" or cmd == "east":
+        if player.current_room.e_to:
+            dir = "Go East!"
+            player.current_room = player.current_room.e_to
+        else:
+            return print('\x1b[0;33;40m' + "I'm sorry you can't go East." + '\x1b[0m')
+    elif cmd == "w" or cmd == "W" or cmd == "West" or cmd == "west":
+        if player.current_room.w_to:
+            dir = "Go West!"
+            player.current_room = player.current_room.w_to
+        else:
+            return print('\x1b[0;33;40m' + "I'm sorry you can't go West." + '\x1b[0m')
+    else:
+        return print('\x1b[0;33;40m' + "I did not understand that command" + '\x1b[0m')
+    return print('\x1b[0;33;40m' + dir + '\x1b[0m' + "\n")
+
```

Comment:
You might be able to refactor this code to make it dryer by utilizing hasattr and getattr ...